### PR TITLE
[DC-1542] integration test for storing pid/rid/shift mappings in primary table

### PR DESCRIPTION
**Pull request by lrwb-aou**
* fixes one bug in the cleaning rule
* mocks the hard coded PIPELINE_TABLES primary dataset reference
* makes sure existing values are not overwritten.
* makes sure new values are added along with random shift elements and partition dates

### CODE REVIEW

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings_test.py
Comment (ksdkalluri) [inline tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings_test.py:8]: looks good!
nit: there are 2 unused Imports in the test script.

**Review by ksdkalluri — APPROVED** [review]

File: tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings_test.py
Comment (lrwb-aou) [reply tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/store_pid_rid_mappings_test.py]: @ksdkalluri nice catch!  Just removed them.  I'll wait until tests pass, and then merge with your approval.

**Pull request merged**
